Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Succulent",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "Succulent",
+            targets: ["Succulent"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/envoy/Embassy.git", .branch("master")),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "Succulent",
+            dependencies: ["Embassy"],
+            path: "./Succulent/Classes",
+            exclude:["NSFileHandle+readLine"]
```

Comment:
Can we remove this line now?

Reply:
You mean exclude:["NSFileHandle+readLine"], right?

Yes, we can remove this line